apps/wallet/src/ui/app/components/receipt-card/SponsoredTxnGasSummary.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiAddress } from '@mysten/sui.js';
+
+import { TxnAddressLink } from './TxnAddressLink';
+import { useFormatCoin } from '_hooks';
+import { GAS_TYPE_ARG } from '_redux/slices/sui-objects/Coin';
+import { Text } from '_src/ui/app/shared/text';
+
+type SponsoredTxnGasSummaryProps = {
+    totalGas: number;
+    sponsor: SuiAddress;
+};
+
+export function SponsoredTxnGasSummary({
+    totalGas,
+    sponsor,
+}: SponsoredTxnGasSummaryProps) {
+    const [senderTotalAmount, senderTotalAmountSymbol] = useFormatCoin(
```

Comment:
why format 0?

Reply:
Discussed this offline - I confirmed that a sponsor always has to pay 100% of the gas for a transaction hence why the sender pays 0 SUI in this case